### PR TITLE
Switch typescript and cpplite editors to multiviews

### PR DESCRIPTION
**Pull request by matthiasblaesing**
I noticed that the "History" tab was missing for typescript editor. At times I use it to access the history
of a single file. I think it is appropriate to use MultiViews for the typescript editor. As the typescript
and cpplite editors are from the same heritage, I adjusted them too.

### CODE REVIEW

File: cpplite/cpplite.editor/src/org/netbeans/modules/cpplite/editor/file/CDataObject.java [hunk 1]
```diff
@@ -112,14 +116,27 @@
 @GrammarRegistration(grammar="resources/c.tmLanguage.json", mimeType=MIMETypes.C)
 public class CDataObject extends MultiDataObject {
 
+    static final String ICON = "org/netbeans/modules/cpplite/editor/file/resources/CSrcIcon.gif";
```
Comment (junichi11) [inline cpplite/cpplite.editor/src/org/netbeans/modules/cpplite/editor/file/CDataObject.java:119]: Maybe, we can add `@StaticResource`.
Other files as well.

Comment (matthiasblaesing) [reply cpplite/cpplite.editor/src/org/netbeans/modules/cpplite/editor/file/CDataObject.java:121]: Valid point - updated this accordingly.

**Comment by matthiasblaesing** [comment]
If noone objects, I'll merge this by the end of the week.

**Review by junichi11 — APPROVED** [review]
Great! Looks good to me. 
(Nitpick: It would be nice if you could add screenshots for before and after.)

**Comment by matthiasblaesing** [comment]
Screenshot after:

Screenshot

The highlighted area is the result of adding multiviews. The editor is the the base view, that was present before, while the history tab get added by this changeset.

**Comment by junichi11** [comment]
Perfect! Thank you for it.

**Pull request merged**